tests/utils/test_cli.py

```diff
@@ -30,3 +32,22 @@ def test_cli_env(self):
         self.assertIn("Python version", cs.out)
         self.assertIn("Platform", cs.out)
         self.assertIn("Using distributed or parallel set-up in script?", cs.out)
+
+    @is_pt_tf_cross_test
```

Comment:
I'm assuming `tests/utils/test_cli.py` is run with `is_pt_tf_cross_test` at least on `self_push` ([here](https://github.com/huggingface/transformers/blob/main/.github/workflows/self-push.yml))